pkg/virt-config/config-map.go

```diff
@@ -34,17 +33,23 @@ import (
 	"kubevirt.io/kubevirt/pkg/util"
 )
 
-const featureGateEnvVar = "FEATURE_GATES"
-
 const (
-	dataVolumesGate   = "DataVolumes"
-	cpuManager        = "CPUManager"
-	liveMigrationGate = "LiveMigration"
-	SRIOVGate         = "SRIOV"
-	configMapName     = "kubevirt-config"
+	configMapName         = "kubevirt-config"
+	featureGateEnvVar     = "FEATURE_GATES"
+	emulatedMachineEnvVar = "VIRT_EMULATED_MACHINES"
 )
 
-func ParseFeatureGatesFromConfigMap() {
+func init() {
```

Comment:
I know we just advised you to put this logic in an init function, but there's a problem with this.

The init function is going to load when the package is imported. This init function assumes that a cluster is available to pull the configmap from, which is not always the cases. For example, in our unit test suite there isn't an active cluster. Does 'make test' pass?

Reply:
argh, too bad... :/
But having to call Load() manually is ugly
But atm I also have no nice idea how to solve this...